openshift_cli_installer/libs/clusters/ocp_cluster.py

```diff
@@ -139,6 +144,19 @@ def _add_s3_bucket_data(self):
             f"{self.cluster_info['name']}-{self.cluster_info['shortuuid']}.zip"
         )
 
+    def check_and_assign_aws_cluster_region(self):
+        if not self.cluster_info.get("region"):
+            if self.cluster_info["platform"] in [AWS_STR, AWS_OSD_STR]:
+                region, vpcs = 0, 0
+                for _region in aws_region_names():
+                    num_vpcs = len(boto3.client(service_name="ec2", region_name=_region).describe_vpcs()["Vpcs"])
+                    if num_vpcs <= vpcs:
```

Comment:
I think `<=` should be `<`.
`=` do not add any value here.

Reply:
Added `<=` because initially `vpcs is 0` and `num_vpcs` can never be less than `0 or vpcs`, so this if condition will never become true for `num_vpcs < vpcs`